### DISCUSSION TITLE
转化det与rec后，测试推理模型没结果

### DISCUSSION THREAD

**Discussion by Advance-Mr** [PaddlePaddle/PaddleOCR#14326] [Q&A]
### 🔎 Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

 1、
E:\PaddleOCR-release-2.6>python tools/infer/predict_system.py --image_dir="C:\Users\Administrator\Desktop\ttt\6.jpg" --det_model_dir="./inference_
model/det/" --rec_model_dir="./inference_model/rec/"
[2024/11/26 11:37:39] ppocr INFO: In PP-OCRv3, rec_image_shape parameter defaults to '3, 48, 320', if you are using recognition model with PP-OCRv2 or an older version, please set --rec_image_shape='3,32,320
[2024/11/26 11:37:41] ppocr DEBUG: dt_boxes num : 18, elapsed : 2.442652463912964
[2024/11/26 11:37:42] ppocr DEBUG: rec_res num  : 18, elapsed : 0.4737098217010498
[2024/11/26 11:37:42] ppocr DEBUG: 0  Predict time of C:\Users\Administrator\Desktop\ttt\6.jpg: 2.982s
[2024/11/26 11:37:42] ppocr DEBUG: The visualized image saved in ./inference_results\6.jpg
[2024/11/26 11:37:42] ppocr INFO: The predict total time is 3.398603916168213
2、

(paddle2) E:\PaddleOCR-release-2.6>python tools/infer_rec.py -c configs/rec/PP-OCRv3/multi_language/chinese_cht_PP-OCRv3_rec.yml -o Global.pretrained_model=./output/v3_chinese_cht_mobile/best_accuracy.pdparams     Global.infer_img="C:\Users\Administrator\Desktop\7.jpg"
[2024/11/26 11:21:23] ppocr INFO: Architecture :
[2024/11/26 11:21:23] ppocr INFO:     Backbone :
[2024/11/26 11:21:23] ppocr INFO:         last_conv_stride : [1, 2]
[2024/11/26 11:21:23] ppocr INFO:         last_pool_type : avg
[2024/11/26 11:21:23] ppocr INFO:         name : MobileNetV1Enhance
[2024/11/26 11:21:23] ppocr INFO:         scale : 0.5
[2024/11/26 11:21:23] ppocr INFO:     Head :
[2024/11/26 11:21:23] ppocr INFO:         head_list :
[2024/11/26 11:21:23] ppocr INFO:             CTCHead :
[2024/11/26 11:21:23] ppocr INFO:                 Head :
[2024/11/26 11:21:23] ppocr INFO:                     fc_decay : 1e-05
[2024/11/26 11:21:23] ppocr INFO:                 Neck :
[2024/11/26 11:21:23] ppocr INFO:                     depth : 2
[2024/11/26 11:21:23] ppocr INFO:                     dims : 64
[2024/11/26 11:21:23] ppocr INFO:                     hidden_dims : 120
[2024/11/26 11:21:23] ppocr INFO:                     name : svtr
[2024/11/26 11:21:23] ppocr INFO:                     use_guide : True
[2024/11/26 11:21:23] ppocr INFO:             SARHead :
[2024/11/26 11:21:23] ppocr INFO:                 enc_dim : 512
[2024/11/26 11:21:23] ppocr INFO:                 max_text_length : 100
[2024/11/26 11:21:23] ppocr INFO:         name : MultiHead
[2024/11/26 11:21:23] ppocr INFO:     Transform : None
[2024/11/26 11:21:23] ppocr INFO:     algorithm : SVTR
[2024/11/26 11:21:23] ppocr INFO:     model_type : rec
[2024/11/26 11:21:23] ppocr INFO: Eval :
[2024/11/26 11:21:23] ppocr INFO:     dataset :
[2024/11/26 11:21:23] ppocr INFO:         data_dir : ./train_data/rec/
[2024/11/26 11:21:23] ppocr INFO:         label_file_list : ['./train_data/rec/val.txt']
[2024/11/26 11:21:23] ppocr INFO:         name : SimpleDataSet
[2024/11/26 11:21:23] ppocr INFO:         transforms :
[2024/11/26 11:21:23] ppocr INFO:             DecodeImage :
[2024/11/26 11:21:23] ppocr INFO:                 channel_first : False
[2024/11/26 11:21:23] ppocr INFO:                 img_mode : BGR
[2024/11/26 11:21:23] ppocr INFO:             MultiLabelEncode : None
[2024/11/26 11:21:23] ppocr INFO:             RecResizeImg :
[2024/11/26 11:21:23] ppocr INFO:                 image_shape : [3, 48, 320]
[2024/11/26 11:21:23] ppocr INFO:             KeepKeys :
[2024/11/26 11:21:23] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2024/11/26 11:21:23] ppocr INFO:     loader :
[2024/11/26 11:21:23] ppocr INFO:         batch_size_per_card : 8
[2024/11/26 11:21:23] ppocr INFO:         drop_last : False
[2024/11/26 11:21:23] ppocr INFO:         num_workers : 4
[2024/11/26 11:21:23] ppocr INFO:         shuffle : False
[2024/11/26 11:21:23] ppocr INFO: Global :
[2024/11/26 11:21:23] ppocr INFO:     cal_metric_during_train : True
[2024/11/26 11:21:23] ppocr INFO:     character_dict_path : ./ppocr/utils/dict/chinese_cht_dict.txt
[2024/11/26 11:21:23] ppocr INFO:     checkpoints : None
[2024/11/26 11:21:23] ppocr INFO:     debug : False
[2024/11/26 11:21:23] ppocr INFO:     distributed : False
[2024/11/26 11:21:23] ppocr INFO:     epoch_num : 400
[2024/11/26 11:21:23] ppocr INFO:     eval_batch_step : [0, 200]
[2024/11/26 11:21:23] ppocr INFO:     infer_img : C:\Users\Administrator\Desktop\7.jpg
[2024/11/26 11:21:23] ppocr INFO:     infer_mode : False
[2024/11/26 11:21:23] ppocr INFO:     log_smooth_window : 20
[2024/11/26 11:21:23] ppocr INFO:     max_text_length : 100
[2024/11/26 11:21:23] ppocr INFO:     pretrained_model : ./output/v3_chinese_cht_mobile/best_accuracy.pdparams
[2024/11/26 11:21:23] ppocr INFO:     print_batch_step : 10
[2024/11/26 11:21:23] ppocr INFO:     save_epoch_step : 10
[2024/11/26 11:21:23] ppocr INFO:     save_inference_dir : None
[2024/11/26 11:21:23] ppocr INFO:     save_model_dir : ./output/v3_chinese_cht_mobile/
[2024/11/26 11:21:23] ppocr INFO:     save_res_path : ./output/rec/chinese_cht.txt
[2024/11/26 11:21:23] ppocr INFO:     use_gpu : True
[2024/11/26 11:21:23] ppocr INFO:     use_space_char : True
[2024/11/26 11:21:23] ppocr INFO:     use_visualdl : False
[2024/11/26 11:21:23] ppocr INFO: Loss :
[2024/11/26 11:21:23] ppocr INFO:     loss_config_list :
[2024/11/26 11:21:23] ppocr INFO:         CTCLoss : None
[2024/11/26 11:21:23] ppocr INFO:         SARLoss : None
[2024/11/26 11:21:23] ppocr INFO:     name : MultiLoss
[2024/11/26 11:21:23] ppocr INFO: Metric :
[2024/11/26 11:21:23] ppocr INFO:     ignore_space : False
[2024/11/26 11:21:23] ppocr INFO:     main_indicator : acc
[2024/11/26 11:21:23] ppocr INFO:     name : RecMetric
[2024/11/26 11:21:23] ppocr INFO: Optimizer :
[2024/11/26 11:21:23] ppocr INFO:     beta1 : 0.9
[2024/11/26 11:21:23] ppocr INFO:     beta2 : 0.999
[2024/11/26 11:21:23] ppocr INFO:     lr :
[2024/11/26 11:21:23] ppocr INFO:         learning_rate : 0.0001
[2024/11/26 11:21:23] ppocr INFO:         name : Cosine
[2024/11/26 11:21:23] ppocr INFO:         warmup_epoch : 5
[2024/11/26 11:21:23] ppocr INFO:     name : Adam
[2024/11/26 11:21:23] ppocr INFO:     regularizer :
[2024/11/26 11:21:23] ppocr INFO:         factor : 3e-05
[2024/11/26 11:21:23] ppocr INFO:         name : L2
[2024/11/26 11:21:23] ppocr INFO: PostProcess :
[2024/11/26 11:21:23] ppocr INFO:     name : CTCLabelDecode
[2024/11/26 11:21:23] ppocr INFO: Train :
[2024/11/26 11:21:23] ppocr INFO:     dataset :
[2024/11/26 11:21:23] ppocr INFO:         data_dir : ./train_data/rec/
[2024/11/26 11:21:23] ppocr INFO:         ext_op_transform_idx : 1
[2024/11/26 11:21:23] ppocr INFO:         label_file_list : ['./train_data/rec/train.txt']
[2024/11/26 11:21:23] ppocr INFO:         name : SimpleDataSet
[2024/11/26 11:21:23] ppocr INFO:         transforms :
[2024/11/26 11:21:23] ppocr INFO:             DecodeImage :
[2024/11/26 11:21:23] ppocr INFO:                 channel_first : False
[2024/11/26 11:21:23] ppocr INFO:                 img_mode : BGR
[2024/11/26 11:21:23] ppocr INFO:             RecConAug :
[2024/11/26 11:21:23] ppocr INFO:                 ext_data_num : 2
[2024/11/26 11:21:23] ppocr INFO:                 image_shape : [48, 320, 3]
[2024/11/26 11:21:23] ppocr INFO:                 prob : 0.5
[2024/11/26 11:21:23] ppocr INFO:             RecAug : None
[2024/11/26 11:21:23] ppocr INFO:             MultiLabelEncode : None
[2024/11/26 11:21:23] ppocr INFO:             RecResizeImg :
[2024/11/26 11:21:23] ppocr INFO:                 image_shape : [3, 48, 320]
[2024/11/26 11:21:23] ppocr INFO:             KeepKeys :
[2024/11/26 11:21:23] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2024/11/26 11:21:23] ppocr INFO:     loader :
[2024/11/26 11:21:23] ppocr INFO:         batch_size_per_card : 8
[2024/11/26 11:21:23] ppocr INFO:         drop_last : False
[2024/11/26 11:21:23] ppocr INFO:         num_workers : 4
[2024/11/26 11:21:23] ppocr INFO:         shuffle : True
[2024/11/26 11:21:23] ppocr INFO: profiler_options : None
[2024/11/26 11:21:23] ppocr INFO: train with paddle 2.6.0 and device Place(gpu:0)
W1126 11:21:23.593571 29304 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 8.6, Driver API Version: 12.3, Runtime API Version: 11.8
W1126 11:21:23.600572 29304 gpu_resources.cc:164] device: 0, cuDNN Version: 8.9.
[2024/11/26 11:21:23] ppocr INFO: load pretrain successful from ./output/v3_chinese_cht_mobile/best_accuracy
[2024/11/26 11:21:23] ppocr INFO: infer_img: C:\Users\Administrator\Desktop\7.jpg
[2024/11/26 11:21:25] ppocr INFO:        result:        0.0
[2024/11/26 11:21:25] ppocr INFO: success!

### 🏃‍♂️ Environment (运行环境)

windows11
paddle 2.60
pycharm
python 3.8

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

另一张结果(paddle2) E:\PaddleOCR-release-2.6>python tools/infer/predict_system.py --image_dir="C:\Users\Administrator\Desktop\3.jpg" --det_model_dir="./inference_model/det/" --rec_model_dir="./inference_model/rec/"
[2024/11/26 11:25:06] ppocr INFO: In PP-OCRv3, rec_image_shape parameter defaults to '3, 48, 320', if you are using recognition model with PP-OCRv2 or an older version, please set --rec_image_shape='3,32,320
[2024/11/26 11:25:08] ppocr DEBUG: dt_boxes num : 4, elapsed : 2.092691421508789
[2024/11/26 11:25:09] ppocr DEBUG: rec_res num  : 4, elapsed : 0.46893882751464844
[2024/11/26 11:25:09] ppocr DEBUG: 0  Predict time of C:\Users\Administrator\Desktop\3.jpg: 2.568s
[2024/11/26 11:25:09] ppocr DEBUG: 肝, 0.531
[2024/11/26 11:25:09] ppocr DEBUG: The visualized image saved in ./inference_results\3.jpg
[2024/11/26 11:25:09] ppocr INFO: The predict total time is 2.586456298828125

**Comment by Advance-Mr** [comment]
换了一个测试方法，提示如下错误
![bdd1c5bff3a0c200088bee6b4adaf1f](https://github.com/user-attachments/assets/d76efac3-af41-4a58-99c9-f8c3af27b6cd)

**Comment by GreatV** [comment]
把识别模型换成官方默认模型试试
